frontend/src/forms/ParcelDetailForm.tsx

```diff
@@ -93,18 +94,36 @@ const ParcelDetailForm = (props: ParcelPropertyProps) => {
             initialValues={initialValues}
             validationSchema={ParcelSchema}
             enableReinitialize
-            onSubmit={(values, { setSubmitting }) => {
+            onSubmit={(values, { setSubmitting, resetForm, setStatus, setErrors }) => {
+              let response: any;
               if (!values.id) {
-                dispatch(createParcel(values));
+                response = dispatch(createParcel(values));
```

Comment:
handling the response here instead of using the network reducer because I want all of the formik data in context.